cmd/app/cluster/create/create.go

```diff
@@ -47,11 +44,12 @@ func NewCreateClusterCommand(l log.Logger) *cobra.Command {
 			}
 
 			createOperatorOptions := &manager.CreateOperatorOptions{
-				OperatorImage:   options.OperatorImage,
 				Namespace:       options.OperatorNamespace,
 				Timeout:         time.Duration(options.Timeout) * time.Second,
 				DryRun:          options.DryRun,
 				OperatorVersion: options.OperatorVersion,
+				ImageRegistry:   options.ImageRegistry,
+				RepoName:        options.RepoName,
```

Comment:
The repo name(or organization name) can be hard code as const variables, it would not change frequently.